### DISCUSSION TITLE
Question About Certifications

### DISCUSSION THREAD

**Discussion by wang975679801** [PaddlePaddle/PaddleSpatial#288] [BaiduKDDCup2022] [answered]
After competition, can we check the ranking in the https://aistudio.baidu.com/aistudio/competition/detail/152/0/leaderboard if our team is out of top15? There is no significant gap between top 20 teams, a lucky ratio may change the ranking significantly. Could organizer provide a little bit more certificants?

**Comment by zhoujingbo** [comment, marked as answer]
We can give a certificate of successful participation to your team if you can submit a report and the code no matter your rank. 

If you want to submit a report, you can send an email request to us before July 19.

**Reply by wang975679801** [reply]
I have sent the email to [kddcup2022@baidu.com]. Thanks a lot.

**Reply by zhoujingbo** [reply]
OK，got it. thanks~

**Reply by zhoujingbo** [reply]
We have sent out the notification. If you do not receive it, please send a request to our email (kddcup2022@baidu.com) again. Thanks.
